packages/kbn-openapi-bundler/src/utils/write_yaml_document.ts

```diff
@@ -31,13 +31,14 @@ function stringifyToYaml(document: unknown): string {
 
     // Disable YAML Anchors https://yaml.org/spec/1.2.2/#3222-anchors-and-aliases
     // It makes YAML much more human readable
-    return dump(clearedDocument, {
+    return safeDump(clearedDocument, {
       noRefs: true,
       sortKeys: sortYamlKeys,
+      skipInvalid: true, // Skip invalid types like `undefined`
```

Comment:
Adding `skipInvalid: true` allows the files to be written even if they contain invalid types (e.g. `undefined`). The result is that the invalid types will not be written to the file rather than the default behavior of throwing an exception.

Previous use of the dump function would write files containing invalid types. It looks like we relied on parsing the file contents to handling removing the invalid values, which may no longer be necessary.